kubetest/main.go

```diff
@@ -110,6 +111,7 @@ type options struct {
 func defineFlags() *options {
 	o := options{}
 	flag.Var(&o.build, "build", "Rebuild k8s binaries, optionally forcing (release|quick|bazel) strategy")
+	flag.BoolVar(&o.buildIngressGCE, "build-ingressgce", "Build ingress-gce binaries")
```

Comment:
s/BoolVar/Var

Reply:
There are no options for that flag so @fejta suggested just making it a bool.

Reply:
Yeah I think this is OK. It can't be a build strategy without being a var, and buildStrategy seems OK to me. Particularly if it might have different ones in the future (bazel vs make + docker?)

Reply:
Ack. Done